test_integration_trello.py

```diff
@@ -0,0 +1,41 @@
+import os
+from dotenv import load_dotenv, find_dotenv
+import flask
+import json
+import requests
+import responses
+import requests_mock
+import pytest
+from unittest.mock import patch, Mock
+import unittest
+from requests_mock_flask import add_flask_app_to_mock
+
+import app
+
+@pytest.fixture
+def client():
+    file_path = find_dotenv('.env.test')
+    load_dotenv(file_path, override=True)
+    test_app = app.create_app()
+    test_app.config['LOGIN_DISABLED'] = True
```

Comment:
You could put this in .env.test instead. Also, does your e2e test without disabling login?